ESM entry uses extensionless relative imports; fails under strict ESM resolution (Rspack/Webpack fullySpecified)

The ESM entry selected by `exports.import` points to `src/js/plyr.js`, which contains extensionless relative imports (e.g. `./utils/urls`). With strict ESM resolution (e.g. Rspack/Webpack’s fully specified mode or Node ESM rules), this fails to resolve and breaks builds.

## Current behaviour

`plyr` publishes the below
```json
"exports": {
    ".": {
        "types": "./src/js/plyr.d.ts",
        "import": "./src/js/plyr.js",
        "require": "./dist/plyr.js",
        "browser": "./dist/plyr.min.js",
        "default": "./dist/plyr.js"
    },
    "./dist/plyr.css": "./dist/plyr.css",
    "./css": "./dist/plyr.css",
    "./plyr.scss": "./src/sass/plyr.scss"
},
```

The `src` uses extensionless relative imports like:

```js
import { parseUrl } from './utils/urls';
```

Modern bundlers enforcing fully specified ESM fail with:

```bash
Module not found: Can't resolve './utils/urls'
help: Did you mean './utils/urls.js'?
The request './utils/urls' failed to resolve only because it was resolved as fully specified...
```

Noted on Version: `plyr@3.8.3` at least

## Expected behaviour
* ESM consumers should be able to build without custom resolver relaxations.
* Either:
    * All relative imports in src include file extensions (e.g. ./utils/urls.js), or
    * The ESM import condition points to a compiled ESM file in dist (e.g. ./dist/plyr.esm.js) with compliant imports.

## Why it matters
Strict ESM resolution requires fully specified file extensions for relative specifiers.

Many modern toolchains (Rspack/Webpack in strict mode, Node ESM) adhere to this and fail on extensionless imports.

# Minimal repro
* Install `plyr@3.8.3`
* Use a bundler that honors fully specified ESM (e.g. Rspack/Webpack 5 defaults when type: "module").
* import 'plyr'
* Build → error as above.

## Workarounds (for consumers)
* Relax resolution for JS: resolve.fullySpecified: false for \.m?js$

Note this is not a great or longterm fix but relaxing this rule or pinning your install version to a known working one will get you past for now. For me the pinned version is currently `3.7.8`

> Note: A few other people have raised this issue so I will try and link them all together.

--- a/package.json
+++ b/package.json
@@ -25,7 +25,7 @@
   "exports": {
     ".": {
       "types": "./src/js/plyr.d.ts",
-      "import": "./src/js/plyr.js",
+      "import": "./dist/plyr.mjs",
       "require": "./dist/plyr.js",
       "browser": "./dist/plyr.min.js",
       "default": "./dist/plyr.js"

--- a/src/js/captions.js
+++ b/src/js/captions.js
@@ -269,7 +269,9 @@ const captions = {
 
       // Handle Vimeo captions
       if (this.isVimeo) {
-        this.embed.enableTextTrack(language);
+        // Enable text track but don't render captions within the player
+        // Since we handle that ourselves
+        this.embed.enableTextTrack(language, null, false);
       }
 
       // Trigger event

--- a/src/sass/components/captions.scss
+++ b/src/sass/components/captions.scss
@@ -44,7 +44,7 @@
 
 .plyr__caption {
   background: $plyr-captions-background;
-  border-radius: 2px;
+  border-radius: 4px;
   box-decoration-break: clone;
   color: $plyr-captions-text-color;
   line-height: 185%;

